### DISCUSSION TITLE
Relocation of the repo

### DISCUSSION THREAD

**Discussion by CM000n** [UI-Lovelace-Minimalist/UI#158] [Q&A] [answered]
Hi guys, I wanted to ask what you think. I have been thinking about this for the last few days and now I see no other option than to change the organisation and the repo in order to remain able to act. It's hard for me because the whole project is Paddy's baby and I don't want to decide anything over his head. But since we haven't heard from him for almost three months now, I think it's time to do something. 
Therefore, if there are no objections from your point of view, I would create a new organisation during the week and move the repo, what do you think?

**Comment by edoardop13** [comment]
I'm agree with you

**Reply by CM000n** [reply]
I wanted to add some of the other contributors of recent times as organisation owners also. Including of course @Paddy0174 . That should avoid a situation like we have now in the future.

**Comment by mp-se** [comment]
It's a great project and would be a pity if it cannot be maintained. Paddy has done some great work but there is more to do. I understand that its not possible to update the wiki and that is not a good thing. I support moving to another reposity and it might be a good idea to split the core and custom cards into different repos to release those independant.

**Reply by CM000n** [reply]
Hey guys, I created a new organization at this address https://github.com/Lovelace-Minimalist

**Reply by CM000n** [reply]
Think I will mirror the existing repo later today or tomorrow

**Comment by Paddy0174** [comment, marked as answer]
Hello everybody!

I' very sorry, I had an accident and wasn't able (was forbidden by my eye doctor) to use any of my modern toys like a computer or phone (more on that at another place). This seems to be the most pressing issue for you, so I just setup @schumijo and @CM000n as Admins for this repo. Hope this solves at least this 

I really am very sorry, believe me, but I have to get a little overview of what changes and contributions have come along in the last weeks. :) 

Please, give me some time, I'll try to read up on everything here. :) 

@CM000n 
I'll send you a PM later today (or tomorrow at the latest), but I have to do some read up. Right now, I'm quite overwhelmed what you and others have done here! Thanks a lot, you're doing a great job here! :) 

See (or better read) you later! :)

**Reply by CM000n** [reply]
Wohoow, that's great news. :-) I am glad that you are doing well, or at least well under the circumstances. I'm glad you're back on board. See (read) you later ;-)

**Reply by bavo** [reply]
Paddy, you’re back ❤️

**Reply by KaherdinTristan** [reply]
Paddy❤️ !! 🤟 🌈 🌻

**Reply by schumijo** [reply]
Paddy ! Happy to see you again !

**Reply by mp-se** [reply]
Really good that you are back, and its great news that you are getting better!

**Reply by fastender** [reply]
Get well soon. We hope that you will regain your strength and enrich us in 2022 as well.

**Reply by CM000n** [reply]
Hello @Paddy0174 , I hope you are well and have recovered a bit in the meantime.
Any new news on when you'll be able to rejoin? We miss you :-)
